scripts/commands/updateApiDocs.ts

```diff
@@ -114,6 +114,47 @@ const readArgs = (): Arguments => {
     .parseSync();
 };
 
+const getLegacyReleaseNotes = async (
+  pkg: Pkg,
+): Promise<{ title: string; url: string }[]> => {
+  /* Check for markdown files in `docs/api/package-name/release-notes/
+   * then sort them and create entries for the TOC.
+   */
+  if (!(pkg.name === "qiskit")) {
+    return [];
+  }
+  await $`mkdir -p ${getRoot()}/docs/api/${pkg.name}/release-notes`.quiet();
+  const legacyReleaseNoteVersions = (
+    await $`ls ${getRoot()}/docs/api/${pkg.name}/release-notes`.quiet()
+  ).stdout
+    .split("\n")
+    .filter((x) => x)
```

Comment:
Is this to remove empty lines? Could you call `stdout.strip()` before to avoid this?